### PR TITLE
Arjun, Ashish | BAH-3849 | Enhancements on Medication Page

### PR DESCRIPTION
**Pull request by binduak**
JIRA card -> [https://bahmni.atlassian.net/browse/BAH-3849](https://bahmni.atlassian.net/browse/BAH-3849)

### CODE REVIEW

File: ui/app/common/domain/services/diagnosisService.js [hunk 1]
```diff
@@ -17,18 +17,21 @@ angular.module('bahmni.common.domain')
...
+            var url = Bahmni.Common.Constants.bahmniDiagnosisUrl;
+            return $http.get(url, {
+                params: { patientUuid: patientUuid }
             });
         };
 
         this.deleteDiagnosis = function (obsUuid) {
             var url = Bahmni.Common.Constants.bahmniDeleteDiagnosisUrl;
             return $http.get(url, {
-                params: {obsUuid: obsUuid}
```
[the remark this answers is not in the document]
Comment (Arjun-Go) [reply ui/app/common/domain/services/diagnosisService.js:27]: This was done to keep things consistent with other service methods. Nothing changes implementation-wise

File: ui/app/styles/clinical/_visit.scss [hunk 2]
```diff
@@ -49,9 +49,7 @@
     }
 }
 
-.opd-wrapper {
```
[the remark this answers is not in the document]
Comment (Arjun-Go) [reply ui/app/styles/clinical/_visit.scss:52]: This was the commit message that I found for this change - `Removed the style as it was affecting the sticky property of child`

[the remark this answers is not in the document]
Comment (rahu1ramesh) [reply ui/app/styles/clinical/_visit.scss:52]: @AshishkurianTw

[the remark this answers is not in the document]
Comment (AshishkurianTw) [reply ui/app/styles/clinical/_visit.scss:52]: So basically this styling was affecting the sticky property for ipd dashboard. There was requirement from cure to keep the patient details placeholder on the ipd dashboard to be sticky when we scroll through the ipd dashboard. This styling was  not  letting it happen . Thats why we removed it. We checked if it was affecting the ui , but when we removed it , everything else working fine . So we went ahead with this change

[the remark this answers is not in the document]
Comment (SanoferSameera) [reply ui/app/styles/clinical/_visit.scss:52]: Ok. Thanks. Going ahead with a merge

**Review by SanoferSameera — APPROVED** [review]

**Pull request merged**